server/controller/modules.js

```diff
@@ -0,0 +1,31 @@
+import models from '../models';
+
+const { Module } = models;
+
+/**
+ * A class that handles modules methods
+ */
+class ModulesController {
+/**
+ * get all modules
+ * @param {object} req - request object
+ * @param {object} res - response object
+ * @param{function} next - next function
+ * @returns {object} response object
+ */
+  static async getAllModules(req, res) {
+    try {
+      const Modules = await Module.findAll();
```

Comment:
let's keep this lowercased `modules` instead of `Modules`

Reply:
Done @fortunee